--- build-openssl-1.0.2d.bat	2015-08-11 21:06:36.587357585 -0300
+++ build-openssl-1.0.2d.bat.patched	2015-08-11 21:12:00.863368853 -0300
@@ -42,7 +42,7 @@
 :: VC Visual Studio version you want to use for building the library.
 
 
-SET VISUAL_STUDIO_VC=D:\dev\vs!VS_VERSION!\VC
+SET VISUAL_STUDIO_VC=C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\
 
 :: Set CYGWIN_DIR to the location of your Cygwin root ( one level up from /bin )
 :: You can use any Cygwin installation, including a portable cygwin: 
@@ -51,7 +51,7 @@
 
 SET CYGWIN_DIR=C:\Cygwin
 
-SET PERL_ROOT=D:\dev\perl
+SET PERL_ROOT=C:\perl
 
 rem ========================================================================================================
 
@@ -87,9 +87,9 @@
 :: Set this to the version of the project you are building
 SET PROJECT_VERSION=1.0.2d
 
-SET PROJECT_SRC_DIR=D:\opensource\!BUILD_PROJECT!-!PROJECT_VERSION!
+SET PROJECT_SRC_DIR=C:\projects\win-openssl\!BUILD_PROJECT!-!PROJECT_VERSION!
 
-SET PROJECT_INSTALL_DIR=D:\dist\!BUILD_PROJECT!-dist-!PROJECT_VERSION!-vs!VS_VERSION!
+SET PROJECT_INSTALL_DIR=C:\projects\win-openssl\!BUILD_PROJECT!-dist-!PROJECT_VERSION!-vs!VS_VERSION!
 
 rem ========================================================================================================
 rem == PLEASE DO NOT EDIT BELOW THIS LINE
@@ -532,7 +532,7 @@
 	echo ----------------------------------------------------------------------------
 	echo.
 
-	bash -c "!SEVENZIP_CMD! !SEVENZIP_CMD_OPTIONS! !COMPRESSED_FILE! !DST_DIST!" 1> nul
+	bash -c "\"!SEVENZIP_CMD!\" !SEVENZIP_CMD_OPTIONS! !COMPRESSED_FILE! !DST_DIST!" 1> nul
 
 	echo Compressing in: !COMPRESSED_FILE!
 
@@ -558,14 +558,6 @@
 
 :buildall
 
-IF NOT EXIST "!PERL_ROOT!\perlenv.bat" (
-	ECHO.
-	call :exitB "Make sure !PERL_ROOT! points to a valid Perl root directory. (perlenv.bat is missing)"
-	goto :eof
-)
-
-call !PERL_ROOT!\perlenv.bat
-
 for %%a in (x86 x64) do (
 	for %%l in (shared static) do (
 		for %%b in (debug release) do (
@@ -592,15 +584,6 @@
 	SET BIT_STR=64
 )
 
-
-IF NOT EXIST "!PERL_ROOT!\perlenv.bat" (
-	ECHO.
-	call :exitB "Make sure !PERL_ROOT! points to a valid Perl root directory. (perlenv.bat is missing)"
-	goto :eof
-)
-
-call !PERL_ROOT!\perlenv.bat
-
 echo:
 echo Building OpenSSL Library [!__ARCH!] [!__LINK!] [!__BUILD!]
 echo:
